week-2/Homework/mandatory/4-sorting-algorithm.js

```diff
@@ -21,9 +21,36 @@ You don't have to worry about making this algorithm work fast! The idea is to ge
 */
 //1)Create the function inside, Think about nested loops
 function sortAges(arr) {
-  //create function here
+  let arr1 = [];
+  for (let i = 0; i < arr.length; i++) {
+    if (Number.isInteger(arr[i])) {
+      arr1.push(arr[i]);
+    }
+  }
+  let arr2 = [];
+  for (let i = 0; i < arr1.length; i++) {
+    for (let j = 1 + i; j < arr1.length; j++) {

```

Comment:
You solved this in an interesting way Fatima!! good work
will try later to share a simplified version 
Excellent